src/media/js/payments.js

```diff
@@ -116,6 +116,8 @@ define('payments',
             fxpay.configure(opt.fxpaySettings);
         }
 
+        console.log(opt.paymentWindow);
```

Comment:
maybe prefix this message like ``log('using custom window?', opt.paymentWindow)``